src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -536,6 +540,22 @@ public void stopsim() {
     elbowArmSim.setInputVoltage(0);
   }
   
+  public Rotation3d getBaseRotation3d() {
+    return new Rotation3d(getBaseAngle(), 0, 0);
+  }
+
+  public Rotation3d getElbowRotation3d() {
+    return new Rotation3d(getElbowAngle(), 0, 0);
+  }
+
+  public Pose3d getBasePose3D() {
+    return new Pose3d(0.008083, -0.173961, 0.20541, getBaseRotation3d());
+  }
+
+  public Pose3d getElbowPose3d() {
+    return new Pose3d(getArmX(), getArmY(), -0.042619, getElbowRotation3d());
```

Comment:
Magic numbers should be constants or at least have comments to explain where they come from

Reply:
numbers are offsets from the CAD model in blender and I added comments explaining them.